src/js/common/schemaform/formState.js

```diff
@@ -6,6 +6,12 @@ function isHiddenField(schema) {
   return !!schema['ui:collapsed'] || !!schema['ui:hidden'];
 }
 
+function get(path, data) {
```

Comment:
Thanks for the insight @jbalboni, unless `path` is available as a string somehow in the invocation scope for this function (in which case regex could provide a slighter faster option), this is the clearest, fastest approach.